components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -199,64 +209,93 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
             savePreferences(currentUserId, abPreferences);
             setFirstChannelName('');
         }
+        const {onNextNavigateTo, onPrevNavigateTo} = this.props;
```

Comment:
I would prefer destructuring to be happening at the beginning of the function body